packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -677,37 +680,75 @@ class ModalBottomSheetRoute<T> extends PopupRoute<T> {
   }
 
   @override
-  Widget buildPage(BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation) {
+  Widget buildPage(BuildContext context, Animation<double> animation,
+      Animation<double> secondaryAnimation) {
+    final GlobalKey bottomSheetKey = GlobalKey();
```

Comment:
This will gets recreated every time, should probably store this as a property in ModalBottomSheetRoute